distbench_busybox.cc

```diff
@@ -154,6 +154,28 @@ absl::Status SaveResultProtoToFileBinary(const std::string &filename,
   return absl::OkStatus();
 }
 
+absl::StatusOr<int64_t> GetTestSequenceTimeout(
+    const distbench::TestSequence &test_sequence) {
+  int64_t accumulated_timeout = 0;
+  for (const auto& test : test_sequence.tests()) {
+    auto maybe_timeout = GetNamedAttributeInt64(test, "test_timeout", -1);
+    if (!maybe_timeout.ok()) {
+      return maybe_timeout;
+    }
+    if (*maybe_timeout == -1) {
+      accumulated_timeout = -1;
```

Comment:
As soon as we get a single test with unspecified timeout we can return absl::GetFlag(FLAGS_max_test_duration)

Reply:
Done; it now return the default on the first unspecified timeout.